sklearn/manifold/_mds.py

```diff
@@ -284,25 +346,33 @@ class MDS(BaseEstimator):
     n_components : int, optional, default: 2
         Number of dimensions in which to immerse the dissimilarities.
 
+    method: string, optional, default: smacof
+        Methods for solving the MDS problem are "smacof" and "svd".
+
```

Comment:
We tend to put the parameters in the order they appear in `__init__`. But since we're making these kwonly args, maybe it doesn't matter. WDYT @jnothman 

Also, this deserves a `.. versionadded` directive.

Reply:
Done